CHANGELOG.md

```diff
@@ -4,14 +4,31 @@ Release notes for Agda version 2.7.0
 Highlights
 ----------
 
+* Mimer, a re-implementation of the "auto" term synthesizer, replaces Agsy.
+
+* New syntax `using x ← e` to bind values on the left-hand-side of a function clause.
+
+* Instance search is more performant thanks to a new indexing structure.
```

Comment:
```suggestion
* Instance search is more performant thanks to a new indexing structure.
  Additionally, users can now control how instances should be selected
  in the case multiple candidates exist.
```

(apologies if the alignment is off, I'm writing on my phone; should be a new sentence in the same paragraph)